Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Search_Endpoints_By_Indicators_README.md

```diff
@@ -0,0 +1,61 @@
+This playbooks searches for different indicators (IP,IPV6,File hashes,Domain) in the crowdstrike falcon console. The output will be all the endpoins found associated with provided indicators. Provided agent id as an input will be excluded from the returned list.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+* CrowdStrike Falcon - Search Endpoints By Hash
+
+### Integrations
+
+* CrowdStrikeFalcon
+* CrowdstrikeFalcon
+
+### Scripts
+
+* IsIntegrationAvailable
+
+### Commands
+
+* cs-falcon-device-count-ioc
+* cs-falcon-device-ran-on
+* endpoint
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| FileSha256 | The SHA256 file hash to search for. |  | Optional |
+| HostId | The ID of the host that originated the detection. |  | Optional |
```

Comment:
the description of the input should be agent IDs to exclude from the retuned results